source/speech/speech.py

```diff
@@ -2595,3 +2610,46 @@ def clearTypedWordBuffer() -> None:
 	complete the word (such as a focus change or choosing to move the caret).
 	"""
 	_curWordChars.clear()
+
+
+def cancelPhoneticDescriptionTimer() -> None:

```

Comment:
All of this new code should be added to a new file `source/speech/phoneticDescriptions.py`, and imported into here.